src/server.rs

```diff
@@ -0,0 +1,61 @@
+use crate::verkle::{get_rlp, decode_block, verification};
+use crate::dot::to_dot;
+use actix_web::{web::{self, Json}, App, HttpServer};
+use crate::types::{VerkleReq, VerkleResp};
+use crate::Config;
+use tokio::process::Command;
+
+async fn get_block_info(data: Json<VerkleReq>) -> Result<Json<VerkleResp>, crate::error::Error> {
+    let block_number = data.into_inner().block_number;
+    let block_rlp = get_rlp(block_number).await?;
+    let block = decode_block(block_rlp.clone())?;
```

Comment:
Nit: you can not to clone here
```suggestion
    let block = decode_block(block_rlp)?;
```